src/dir_contents.ml

```diff
@@ -255,6 +266,10 @@ let load_text_files sctx ft_dir
   let generated_files =
     List.concat_map stanzas ~f:(fun stanza ->
       match (stanza : Stanza.t) with
+      | Coq.T _coq ->
+        (* Format.eprintf "[coq] generated_files called at sctx: %a@\n%!" Path.pp File_tree.Dir.(path ft_dir); *)
+        (* FIXME: Need to generate ml files from mlg ? *)
```

Comment:
don't forget this part :)

Reply:
I was planning to tackle this a part of #1555 where we add support building coq plugins, however this may be the wrong place to do it as this should affect the regular `library` stanza, will remove it for now I think.